meteor.bat

```diff
@@ -0,0 +1,21 @@
+@echo off
+
+rem only if we are running from a checkout
```

Comment:
It is weird that batch scripts use `rem` for comments. I don't capitalize `REM` because it shouts for attention and comments don't really need that. Dunno, we don't have batch/powershell style-guide, sadly.